content/api/hub/template_openaichat_ts.md

```diff
@@ -0,0 +1,84 @@
+title = "OpenAI Chat API TS"
+template = "render_hub_content_body"
+date = "2023-07-24T02:00:00Z"
+content-type = "text/html"
+tags = ["openai", "chatgpt", "api", "config", "ts", "typescript", "http"]
+
+[extra]
+author = "teknicus"
+type = "hub_document"
+category = "Template"
```

Comment:
```suggestion
category = "Sample"
```

This would be a sample as a user will not be able to `spin template install ...` this. if you would like to have. it as a template, you would need to add a template folder in the root of the repo and follow a structure similar to that you can find in https://github.com/fermyon/spin-js-sdk/tree/main/templates

Reply:
Thanks for catching that @karthik2804 

Do you think it needs to be a template or a sample?